### DISCUSSION TITLE
Questions about future commercialization and positioning

### DISCUSSION THREAD

**Discussion by dixtdf** [yume-chan/tango-feedback#25] [Q&A] [answered]
### Pre-issue checklist

- [X] I've searched the issue list but couldn't find the feature I want

### How would you like Tango to be improved?

Will a permanent pricing plan be adopted? It's very anxious that a software cannot be bought out, but from my personal perspective, I prefer a permanent license

### How do you think it should work?

null

**Comment by yume-chan** [comment, marked as answer]
As a Web app, we don't think a bought-out pricing is suitable.

The domain name, the hosting server, all continuously costs money to maintain. The main feature of Pro plan is device sharing, which also requires a server to work.

A bought-out/lifetime plan is really a fake promise, and we don't want to make.

**Reply by dixtdf** [reply]
I respect your choice, it would be even better if there was a permanent authorizationヾ(•ω•`)o
